octomap:
    pointcloud_topic: "/semantic_pcl/semantic_pcl"
    world_frame_id: "map"
    resolution: 0.25
    max_range: 15.0
    raycast_range: 10.0
    clamping_thres_min: 0.0001
    clamping_thres_max: 0.9999
    occupancy_thres: 0.5
    prob_hit: 0.7
    prob_miss: 0.4
    psi: 1
    phi: -0.1
    publish_2d_map: true
    min_ground_z: 0.2
    max_ground_z: 1.2
    save_path: /home/hx/iros_code_pub_ws/map/map.ot
